airflow/cli/cli_config.py

```diff
@@ -1100,6 +1100,12 @@ class GroupCommand(NamedTuple):
 CLICommand = Union[ActionCommand, GroupCommand]
 
 DAGS_COMMANDS = (
+    ActionCommand(
+        name="details",
+        help="Get DAG details given a DAG id",
+        func=lazy_load_command("airflow.cli.commands.dag_command.dag_details"),
+        args=(ARG_DAG_ID, ARG_SUBDIR, ARG_OUTPUT, ARG_VERBOSE),
```

Comment:
Is `ARG_SUBDIR` needed for this command? Doesn't seem like it's used, or if the intent is to be able to also specify the subdirectory to find a DAG, `airflow.utils.cli.get_dags()` should be used in the command. Seems like the intent to require a single DAG ID though.

Reply:
I am not aware of the usage of ARG_SUBDIR, just included it because all other commands dealing with a specific DAG ID seemed to have it, should I remove it?

Reply:
Thanks for the insightful response, I've made the change.